configure.ac

```diff
@@ -10,7 +10,7 @@ AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_AUX_DIR([config])
 AC_CONFIG_MACRO_DIR([m4])
 
-AM_INIT_AUTOMAKE([foreign tar-pax])
+AM_INIT_AUTOMAKE([foreign tar-pax subdir-objects])
```

Comment:
Ok.  It was just throwing a ton of warnings at me.  I've never submitted a merge request before.  What happens now?  Should I change it?